app/javascript/controllers/tooltip_controller.js

```diff
@@ -0,0 +1,82 @@
+import { Controller } from '@hotwired/stimulus'
+import {
+  computePosition,
+  flip,
+  shift,
+  offset,
+  arrow
+} from '@floating-ui/dom';
+
+export default class extends Controller {
+  static targets = ["element", "tooltip"];
+  static values = {
+    placement: String,
+    offset: Number,
+    crossAxis: Number
+  };
+
+  initialize() {
+    this.placementOption = this.placementValue || "top"
+    this.offsetOption = this.offsetValue || 10
+    this.crossAxisOption = this.crossAxisValue || 0
+    this.alignmentAxisOption = this.alignmentValue || null
+    this.arrowElement = document.querySelector('#arrow');
+  }
+
+  connect() {
+    this.elementTarget.addEventListener("mouseenter", this.showTooltip);
+    this.elementTarget.addEventListener("mouseleave", this.hideTooltip);
+    this.elementTarget.addEventListener("focus", this.showTooltip);
+    this.elementTarget.addEventListener("blur", this.hideTooltip);
+  }
+
+  showTooltip = () => {
+    tooltip.style.display = 'block';
+    this.#update();
+  }
+
+  hideTooltip = () => {
+    tooltip.style.display = '';
+  }
+
+  hideAllElements = () => {
+    document.querySelectorAll('#tooltip').forEach(element => element.style.display = '');
+  }
```

Comment:
Think we can remove this method.  When the controller disconnects it should wipe out any tooltip content.

Reply:
[32ed87a](https://github.com/maybe-finance/maybe/pull/1065/commits/32ed87a1ec0fbc4bd74a77dda631625f7dceb3fe)